src/pages/index.js

```diff
@@ -3,19 +3,21 @@ import React from "react";
 import Layout from "@narative/gatsby-theme-novela/src/components/Layout";
 import Section from "@narative/gatsby-theme-novela/src/components/Section";
 import SEO from "@narative/gatsby-theme-novela/src/components/SEO";
-import Headings from "@narative/gatsby-theme-novela/src/components/Headings";
+import Heading from "../@narative/gatsby-theme-novela/sections/heading/Heading";
+import SiapaKami from "../@narative/gatsby-theme-novela/sections/siapakami/Siapa.Kami";
 
-function Homes() {
+function Index() {
```

Comment:
Type file must be consistent, change it to Typescript.

Reply:
Already change the file type to `.tsx`